CMakeLists.txt

```diff
@@ -86,6 +86,10 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
     "MinSizeRel" "RelWithDebInfo")
 endif()
 
```

Comment:
Regarding this overall commit: can you clarify how package support helps including TBB in other cmake based packages?

The standard way this project is generally used is via a simple ``add_subdirectory`` command. This ensures that TBB is built along with the main program, and that everything is compiled in the right order to satisfy dependencies. I have never observed non-robust behavior on any platform (linux, macos, windows).